superset/utils/core.py

```diff
@@ -54,6 +54,12 @@
 
 JS_MAX_INTEGER = 9007199254740991   # Largest int Java Script can handle 2^53-1
 
+sources = {
```

Comment:
nit:I think it would be better to make this a function
```
def get_sources_id(source):
    """
    Mapping for source to id for DB API Mutator
    """
    return {
        'dashboard': 0,
        'explore_view': 1,
        'sql_lab': 2,
    }[source]
```

Reply:
I see what you're saying. I was thinking of this more as an Enum in Java so I made the callers set none as the default instead.